src/crypto/openssl/cose_sign.h

```diff
@@ -2,19 +2,69 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
+#include "crypto/openssl/key_pair.h"
+
 #include <openssl/ossl_typ.h>
 #include <span>
 #include <string>
+#include <t_cose/t_cose_sign1_sign.h>
 #include <unordered_map>
 
 namespace ccf::crypto
 {
+  // Standartised field: algorithm used to sign
```

Comment:
```suggestion
  // Standardised field: algorithm used to sign
```

A link to the RFC/IANA registry would be nice to have here